server/form-pages/utils/index.test.ts

```diff
@@ -1,87 +1,65 @@
 /* eslint-disable max-classes-per-file */
 import 'reflect-metadata'
-import {
-  applyYesOrNo,
-  yesOrNoResponseWithDetail,
-  yesNoOrDontKnowResponseWithDetail,
-  getTask,
-  getSection,
-} from './index'
+import * as utils from './index'
```

Comment:
I think we should stick to named imports as it's what we use elsewhere

Reply:
I needed to do a wildcard import here to allow me to use a spy later on. Will tweak the commit message to make it clear (might even add a comment)